src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -0,0 +1,283 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { Component } from 'react'
+import { deepEquals, optionalToString } from 'services/utils'
+import {
+  StyledTokenRelationshipType,
+  StyledLabelToken,
+  StyledInspectorFooterRow,
+  StyledInspectorFooterRowListPair,
+  StyledInspectorFooterRowListKey,
+  StyledInspectorFooterRowListValue,
+  StyledInlineList,
+  StyledDetailsStatusBar,
+  StyledDetailsStatus,
+  StyledDetailsStatusContents,
+  StyledInspectorFooterRowListPairAlternatingRows,
+  StyledInspectorClipboardCopyAll,
+  StyledInspectorFooterRowListKeyValuePair
+} from './styled'
+import ClickableUrls from '../../../components/ClickableUrls'
+import numberToUSLocale from 'shared/utils/number-to-US-locale'
+import { StyledTruncatedMessage } from 'browser/modules/Stream/styled'
+import { Icon, Popup } from 'semantic-ui-react'
+import ClipboardCopier from 'browser-components/ClipboardCopier'
+
+const mapItemProperties = (itemId: string, itemProperties: any) => {
+  if (!itemProperties.length) {
+    return null
+  }
+  const clipboardCopy = (textToCopy: string) => (
+    <div style={{ marginLeft: 'auto' }}>
+      <ClipboardCopier textToCopy={textToCopy} iconSize={10} />
+    </div>
+  )
+
+  const itemPropertiesIncludingId = [
+    { key: '<id>', value: `${itemId}` },
+    ...itemProperties
+  ]
+
+  const allItemProperties: string[] = []
+
+  const sortedItemProperties = () => {
+    return itemPropertiesIncludingId.sort(
+      ({ key: keyA }: any, { key: keyB }: any) =>
+        keyA < keyB ? -1 : keyA === keyB ? 0 : 1
+    )
+  }
+  sortedItemProperties().map((prop: any) => {
+    allItemProperties.push(`${prop.key}: ${prop.value}`)
+  })
+
+  const mappedItemProperties = () => {
+    return sortedItemProperties().map((prop: any, i: any) => {
+      return (
+        <StyledInspectorFooterRowListPairAlternatingRows
+          className="pair"
+          key={'prop' + i}
+          isFirstOrEvenRow={(i + 1) % 2 === 0}
+        >
+          <Popup
+            on="hover"
+            basic
+            wide
+            trigger={
+              <StyledInspectorFooterRowListKeyValuePair>
+                <StyledInspectorFooterRowListKey className="key">
+                  {prop.key + ': '}
+                </StyledInspectorFooterRowListKey>
+                <StyledInspectorFooterRowListValue className="value">
+                  <ClickableUrls text={optionalToString(prop.value)} />
+                </StyledInspectorFooterRowListValue>
+              </StyledInspectorFooterRowListKeyValuePair>
+            }
+            mouseEnterDelay={1000}
+            hoverable
+            // hideOnScroll
+            popperModifiers={{
```

Comment:
I tried this to fix the type popup issue but no success when I tried - can remove this `popperModifiers` property. Also, can decide if you want to uncomment or remove the `hideOnScroll` on line 95